app/package.json

```diff
@@ -1,23 +1,41 @@
 {
-  "name": "opg-modernising-lpa",
-  "version": "0.0.1",
-  "scripts": {
-    "build": "yarn build:js && yarn build:css && yarn build:images && yarn build:fonts",
-    "build:js": "esbuild --bundle web/assets/main.js --minify --outdir=web/static/javascript --sourcemap --resolve-extensions=.mjs,.js",
-    "build:css": "sass --load-path . --style compressed web/assets/main.scss web/static/stylesheets/all.css",
-    "build:images": "mkdir -p web/static/assets/images && cp node_modules/govuk-frontend/govuk/assets/images/* node_modules/@ministryofjustice/frontend/moj/assets/images/* web/static/assets/images",
-    "build:fonts": "mkdir -p web/static/assets/fonts && cp node_modules/govuk-frontend/govuk/assets/fonts/* web/static/assets/fonts",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "license": "MIT",
-  "dependencies": {
-    "@ministryofjustice/frontend": "^1.4.2",
-    "dart-sass": "^1.25.0",
-    "govuk-frontend": "^4.2.0",
-    "jquery": "^3.6.0"
-  },
-  "devDependencies": {
-    "esbuild": "^0.14.53",
-    "sass": "^1.54.1"
-  }
+    "name": "opg-modernising-lpa",
+    "version": "0.0.1",
+    "scripts": {
+        "build": "yarn build:js && yarn build:css && yarn build:images && yarn build:fonts",
+        "build:js": "esbuild --bundle web/assets/main.js --minify --outdir=web/static/javascript --sourcemap --resolve-extensions=.mjs,.js",
+        "build:css": "sass --load-path . --style compressed web/assets/main.scss web/static/stylesheets/all.css",
+        "build:images": "mkdir -p web/static/assets/images && cp node_modules/govuk-frontend/govuk/assets/images/* node_modules/@ministryofjustice/frontend/moj/assets/images/* web/static/assets/images",
+        "build:fonts": "mkdir -p web/static/assets/fonts && cp node_modules/govuk-frontend/govuk/assets/fonts/* web/static/assets/fonts",
+        "cypress:open": "cypress open",
+        "cypress:run": "cypress run",
+        "cypress:run:debug": "cypress run --headed",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "codecept": "codeceptjs run --steps",
+        "codecept:headless": "HEADLESS=true codeceptjs run --steps",
+        "codecept:ui": "codecept-ui --app",
+        "codecept:demo": "codeceptjs run --steps -c node_modules/@codeceptjs/examples",
+        "codecept:demo:headless": "HEADLESS=true codeceptjs run --steps -c node_modules/@codeceptjs/examples",
+        "codecept:demo:ui": "codecept-ui --app  -c node_modules/@codeceptjs/examples"
```

Comment:
Leaving these here for the time being so others can see what more involved tests look like